dspace-server-webapp/src/main/java/org/dspace/app/rest/repository/ResourcePolicyRestRepository.java

```diff
@@ -52,9 +95,211 @@ public ResourcePolicyRest findOne(Context context, Integer id) {
+            if (action != null) {
+                int actionId = Constants.getActionID(action);
+                resourcePolisies = resourcePolicyService.findByResouceUuidAndActionId(context, resourceUuid, actionId,
+                        Math.toIntExact(pageable.getOffset()),
+                        Math.toIntExact(pageable.getOffset() + pageable.getPageSize()));
+                total = resourcePolicyService.countByResouceUuidAndActionId(context, resourceUuid, actionId);
+            } else {
+                resourcePolisies = resourcePolicyService.findByResouceUuid(context, resourceUuid,
+                        Math.toIntExact(pageable.getOffset()),
+                        Math.toIntExact(pageable.getOffset() + pageable.getPageSize()));
+                total = resourcePolicyService.countByResourceUuid(context, resourceUuid);
+            }
+        } catch (SQLException e) {
+            throw new RuntimeException(e.getMessage(), e);
+        }
+        return converter.toRestPage(resourcePolisies, pageable, total, utils.obtainProjection(true));
+    }
+
+    @PreAuthorize("hasPermission(#epersonUuid, 'EPERSON', 'READ')")
+    @SearchRestMethod(name = "eperson")
+    public Page<ResourcePolicyRest> eperson(@Parameter(value = "uuid", required = true) UUID epersonUuid,
+                                @Parameter(value = "resource", required = false) UUID resourceUuid, Pageable pageable) {
+
+        List<ResourcePolicy> resourcePolisies = null;
+        int total = 0;
+        try {
+            Context context = obtainContext();
+            EPerson eperson = epersonService.find(context, epersonUuid);
+            if (eperson == null) {
+                return null;
+            }
+            if (resourceUuid != null) {
+                resourcePolisies = resourcePolicyService.findByEPersonAndResourceUuid(context, eperson, resourceUuid,
+                        Math.toIntExact(pageable.getOffset()),
+                        Math.toIntExact(pageable.getOffset() + pageable.getPageSize()));
+                total = resourcePolicyService.countResourcePoliciesByEPersonAndResourceUuid(context,
+                        resourceUuid, eperson);
+            } else {
+                resourcePolisies = resourcePolicyService.findByEPerson(context, eperson,
+                        Math.toIntExact(pageable.getOffset()),
+                        Math.toIntExact(pageable.getOffset() + pageable.getPageSize()));
+                total = resourcePolicyService.countByEPerson(context, eperson);
+            }
+        } catch (SQLException e) {
+            throw new RuntimeException(e.getMessage(), e);
+        }
+        return converter.toRestPage(resourcePolisies, pageable, total, utils.obtainProjection(true));
+    }
+
+    @PreAuthorize("hasPermission(#groupUuid, 'GROUP', 'READ')")
+    @SearchRestMethod(name = "group")
+    public Page<ResourcePolicyRest> group(@Parameter(value = "uuid", required = true) UUID groupUuid,
```

Comment:
Again, should have Javadocs.  Again, I'd recommend naming this `findByGroup` as it's finding ResourcePolicies that match a given Group.